src/main/resources/default/assets/scripts/blob-upload-url-popover.js

```diff
@@ -0,0 +1,139 @@
+        buildCancelButtonCol(body);
+        buildErrorCol(body);
+        buildBodyContainer(body);
+        return body;
+    }
+
+    function buildInputFormGroupCol(body) {
+        body.inputLabel = document.createElement('label');
+        body.inputLabel.innerHTML = i18n.label;
+
+        body.inputField = document.createElement('input');
+        body.inputField.classList.add('form-control');
+        body.inputField.setAttribute('type', 'text');
+        body.inputField.setAttribute('placeholder', 'https://');
+
+        body.formGroup = document.createElement('div');
+        body.formGroup.classList.add('form-group');
+        body.formGroup.append(body.inputLabel, body.inputField);
+
+        body.inputCol = document.createElement('div');
+        body.inputCol.classList.add('col', 'col-12');
+        body.inputCol.appendChild(body.formGroup);
+    }
+
+    function buildOkButtonCol(body) {
+        body.buttonOk = document.createElement('button');
+        body.buttonOk.classList.add('btn', 'btn-block', 'btn-primary', 'btn-apply');
+        body.buttonOk.innerHTML = i18n.ok + ' <i class="fa-solid fa-check"></i>';
+        body.buttonOk.addEventListener('click', function (event) {
+            event.preventDefault();
+            updateURL(body.inputField.value);
+        });
+
+        body.leftCol = document.createElement('div');
+        body.leftCol.classList.add('col', 'col-12', 'col-md-6');
+        body.leftCol.appendChild(body.buttonOk);
+    }
+
+    function buildCancelButtonCol(body) {
+        body.buttonCancel = document.createElement('div');
+        body.buttonCancel.classList.add('btn', 'btn-block', 'btn-close', 'btn-outline-secondary');
+        body.buttonCancel.innerHTML = i18n.cancel + ' <i class="fa-solid fa-close"></i>';
+        body.buttonCancel.addEventListener('click', function () {
+            $(triggerElement).popover('toggle');
+        });
+        body.rightCol = document.createElement('div');
+        body.rightCol.classList.add('col', 'col-12', 'col-md-6');
+        body.rightCol.appendChild(body.buttonCancel);
+    }
+
+    function buildErrorCol(body) {
+        body.errorCol = document.createElement('div');
+        body.errorCol.classList.add('col', 'col-12');
+    }
+
+    function buildBodyContainer(body) {
+        body.container = document.createElement('div');
+        body.container.classList.add('row');
+        body.container.append(body.inputCol, body.errorCol, body.leftCol, body.rightCol);
+    }
+
+    /**
+     * Updates the image in the upload box and the url in the form.
+     *
+     * @param url the new URL
+     */
+    const updateURL = function (url) {
+        if (url.startsWith('http://') || url.startsWith('https://')) {
+            let img = uploadContainer.querySelector('.img-preview img');
+            img.setAttribute('src', url);
+            img.onerror = function() {
+                img.setAttribute('src', '/assets/frontend/defaulticons/default-item.png');
```

Comment:
This is an asset from SellSite, maybe use `/assets/images/blob_image_failed.png` instead